cmd/snap/cmd_pack.go

```diff
@@ -37,7 +37,8 @@ type packCmd struct {
 
 var shortPackHelp = i18n.G("Pack the given target dir as a snap")
 var longPackHelp = i18n.G(`
-The pack command packs the given snap-dir as a snap.`)
+The pack command packs the given snap-dir as a snap.
```

Comment:
`The directory must contain a valid meta/snap.yaml file. If the path is not provided, the command operates on/in (?) current directory.`

Reply:
why don't we have `pack` guess the same as `try` does?

Reply:
Not sure I follow, can you elaborate?

Reply:
> If snap-dir argument is omitted, the try command will attempt to infer it if
either snapcraft.yaml file and prime directory or meta/snap.yaml file can be
found relative to current working directory.

Reply:
Aah ok, sounds good!

Reply:
At least two reasons:

1. It has two arguments, not one. We cannot avoid the first argument when we need the second.
2. If we default both to the current directory, the next snap created will include a snap file on it.